index.js

```diff
@@ -593,7 +613,7 @@ class KeyringController extends EventEmitter {
   //
   // Deallocates all currently managed keyrings and accounts.
   // Used before initializing a new vault.
-  async clearKeyrings () {
+  clearKeyrings () {
```

Comment:
Hmm. This seems OK in this case, but I think we should disable the rule that prompted this change (`require-await`).

I thought for a moment that this was `eth-simple-keyring`, and it's important for the keyrings to all share the same API, which means they should all be `async` even if they don't actually do anything async (because another keyring might require `async` for that same method).

Reply:
Disabled the rule inline; I think it's a good rule in general. I considered overriding it in the config to a warning, but that's not really helpful for future generations.